app/views/shared/_header_navbar.html.erb

```diff
@@ -1,4 +1,8 @@
 <header class="lux">
+  <banner dismissible>
```

Comment:
Can you get rid of the dismissible attribute? Then we should be good.

Reply:
Removed.